pennylane/tape/interfaces/autograd.py

```diff
@@ -26,13 +26,13 @@
 
 
 class AutogradInterface(AnnotatedQueue):

```

Comment:
Minor, but I wonder with the new structure if this can be changed to inherit from `QuantumTape`. Not priority to change now, I'll add it to my todo list to think about!

Reply:
That would be a nice side effect. Worth waiting with merging? This may change the separation of the two classes?

Reply:
I don't think this really changes anything too much, it will just make the inheritence less 'surprising' to developers. It's a side effect of Python mixin classes; they have to have the same structure as the class you combine them with, hence why the interfaces need to inherit from `AnnotatedQueue`.